<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="ja" lang="ja">
<head>
<meta name="google-site-verification" content="pH9acLwNugfMheN87eG0dp7noOM1OtNDL4zEeNQfCM8" />
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<meta http-equiv="Content-Style-Type" content="text/css" />
<meta http-equiv="Content-Script-Type" content="text/javascript" />
<title>Daiki Kumakura HOME</title>
<meta name=”robots” content=”index”>
<meta name="author" content="Daiki Kumakura">
<meta name="keywords" content="Daiki Kumakura, 熊倉大騎" />
<meta name="description" content="Daiki Kumakura HOME" />
<link rel="stylesheet" href="index.css" type="text/css" />
</head>
<body>
<div id="outer">
<div id="header"><div id="head-inner"><h1>Daiki Kumakura HOME</h1>
<p class="description"><font size="3"> ----- Daiki KUMAKURA English HOME ----- </font></p></div>
</div>
<div id="contents">

<div class="box4">
	<p>
  		<a href="japanese.html"> <font size="2"> <b>Japanese Page(日本語HOME)</b> </font> </a>
	</p>
</div>


<p>&ensp;</p>


<h2>CONTENTS</h2>

<p>
	■<a href="#CV"> CV</a>
</p>

<p>
	■<a href="#RESEARCH"> RESEARCH</a>
</p>

<p>
	■<a href="#PUBLICATIONS"> PUBLICATIONS</a>
</p>

<p>
	■<a href="#CONTACT"> CONTACT</a>
</p>

<p>
	■<a href="#LINK"> LINK</a>
</p>

<p>
	■<a href="#ARTICLES"> ARTICLES</a>
</p>


<p>&ensp;</p>
<p>&ensp;</p>


<h2 id="CV">CV</h2>

<p> <font size="4" color="#20b2aa"> <strong>EDUCATION</strong> </font> </p>

<p> <font size="4"> <b>~2020</b> </font> </p>
<p>BS：Fisheries Science, Faculty of Fisheries Science, Hokkaido University, Japan</p>

<p> <font size="4"> <b>~2022</b> </font> </p>
<p>MS：Life Science, Graduate School of Life Science, Hokkaido University, Japan</p>

<p> <font size="4"> <b>~Current</b> </font> </p>
<p>Doctoral course：Life Science, Graduate School of Life Science, Hokkaido University, Japan</p>
<p>RIKEN JRA Program student：RIKEN iTHEMS</p>


<p>&ensp;</p>

<p> <font size="4" color="#20b2aa"> <strong>EDUCATIONAL ACTIVITIES</strong> </font> </p>

<p> <font size="4"> <b>2020</b> </font> </p>
<p>[1] Exercise on Bioinformatics, SCI_BIOMOL 3422</p>

<p> <font size="4"> <b>2021</b> </font> </p>
<p>[2] Sustainable Development Goals (SDGs) and Life Science, GEN_LIB 1000 </p>


<p>&ensp;</p>
<p>&ensp;</p>


<h2 id="RESEARCH">RESEARCH</h2>

<p> <font size="4"> <b>・Estimation of transition for microbial community assembly.</b> </font> </p>

<p> <font size="4"> <b>・Search for pathogen of Apis mellifera.</b> </font> </p>

<p> <font size="4"> <b>・Bacterial growth analysis considering sub-population.</b> </font> </p>

<p> <font size="4"> <b>・Social microbiome through hot spring.</b> </font> </p>


<p>&ensp;</p>
<p>&ensp;</p>


<h2 id="PUBLICATIONS">PUBLICATIONS</h2>

<p> <font size="4" color="#20b2aa"> <strong>Peer reviewed papers</strong> </font> </p>

<p> <font size="4"> <b>2020</b> </font> </p>
<p>[1] Tanaka M, <b> Kumakura D </b>, Mino S, Doi H, Ogura Y, Hayashi T, et al. Genomic characterization of closely related species in the Rumoiensis clade infers ecogenomic signatures to non-marine environments. Environ Microbiol. 2020 Aug;22(8):3205–17.</p>

<p> <font size="4"> <b>2021</b> </font> </p>
<p>[2] <b> Kumakura D </b>, Nakaoka S. Exploring Similarity Between Embedding Dimension of Time-Series Data and Flows of an Ecological Population Model. In: Methods of Mathematical Oncology. 2021. Springer Singapore, 69–80. </p>
	
<p> <font size="4"> <b>2022</b> </font> </p>
<p>[3] Komatsu Y, <b> Kumakura D </b>, Seto N, Izumi H, Takeda Y, Ohnishi Y, et al. Dynamic Associations of Milk Components With the Infant Gut Microbiome and Fecal Metabolites in a Mother–Infant Model by Microbiome, NMR Metabolomic, and Time-Series Clustering Analyses. Frontiers in Nutrition. 2022;8: 1192.</p>
<p>[4] Suzuki K, Abe MS, <b>Kumakura D</b>, Nakaoka S, Fujiwara F, Miyamoto H, Nakaguma T, Okada M, Sakurai K, Shimizu S, Iwata H, Masuya H, Nihei N, Ichihashi Y. 2022. Chemical-Mediated Microbial Interactions Can Reduce the Effectiveness of Time-Series-Based Inference of Ecological Interaction Networks. International journal of environmental research and public health 19:1228.</p>

<p>&ensp;</p>

<p> <font size="4" color="#20b2aa"> <strong>Conference & Symposium Presentations</strong> </font> </p>

<p> <font size="4"> <b>2020</b> </font> </p>
<p>[1] <b>Daiki Kumakura</b>, Ryo Yamaguchi, Shinji Nakaoka. Verification of inference of causality method using relative abundance in the Lotka-Volterra equation. JSMB 2020, Japan. September 2020 (oral, japanese)</p>
<p>[2] <b>Daiki Kumakura</b>, Ryo Yamaguchi, Shinji Nakaoka. Reconstruction of a vector field using a time-series causal approach. Applied Mathematics Joint Research Conference, Japan. December 2020 (oral, japanese)</p>
<p> <font size="4"> <b>2021</b> </font> </p>
<p>[3] Shinji Nakaoka, <b>Daiki Kumakura</b>. Application of mathematical science methods for community function analysis and design of soil microbial ecosystems. Soil Microbe × Nutrient Cycling Workshop, Japan. January 2021 (<b>invited</b>, oral, japanese)</p>
<p>[4] <b>Daiki Kumakura</b>. Prediction and availability of ecosystem phenomena at the macro level by optimality of embedding dimension. Soil Microbe × Nutrient Cycling Workshop, Japan. January 2021 (poster, japanese)</p>
<p>[5] <b>Daiki Kumakura</b>, Ryo Yamaguchi, Shinji Nakaoka. Compositional data applicability of causal estimation methods using CCM. ESJ 68, Japan. March 2021 (oral, japanese)</p>
<p>[6] <b>Daiki Kumakura</b>, Ryo Yamaguchi, Shinji Nakaoka. Analysis and validation of microbial growth dynamics with the environment suitability. WAKATE COLOSSEUM FOR BACTERIOLOGY 15th, Japan. August 2021 (oral, japanese)</p>
<p>[7] <b>Daiki Kumakura</b>, Ryo Yamaguchi, Shinji Nakaoka. Analysis and validation of microbial growth dynamics with the environment suitability. WAKATE COLOSSEUM FOR BACTERIOLOGY 15th, Japan. August 2021 (poster, japanese)</p>
<p>[8] <b>Daiki Kumakura</b>, Ryo Yamaguchi, Shinji Nakaoka. Analysis and validation of microbial growth dynamics with the environment suitability. JSMB 2021, Japan. September 2021 (oral, japanese)</p>
<p>[9] <b>Daiki Kumakura</b>, Junya Sunagawa, Takuya Shiramata, Shinji Nakaoka, Masaki Kamakura. Multi-omics analysis of honey bees for searching diseases. JSME2021, Japan. November 2021 (poster, japanese)</p>
<p>[10] Junya Sunagawa, <b>Daiki Kumakura</b>, Takuya Shiramata, Shinji Nakaoka, Masaki Kamakura. Virome analysis to explore virus ubiquity in honeybees. JSME2021, Japan. November 2021 (poster, japanese)</p>
<p>[11] <b>Daiki Kumakura</b>. Microbial dark matter estimation on time series causality method. NetworkScienceResearch2021, Japan. December 2021 (<b>invited</b>, oral, japanese)</p>
<p>[12] <b>Daiki Kumakura</b>. Bacterial growth with subpopulation. Mathematical analysis research for life science workshop 1st, Japan. December 2021 (oral, japanese)</p>
<p> <font size="4"> <b>2022</b> </font> </p>
<p>[13] <b>Daiki Kumakura</b>, Ryo Yamaguchi, Shinji Nakaoka. Growth dynamics of microbes in consideration of environmental suitability. ESJ 69, Japan. March 2022 (oral, japanese)</p>
<p>[14] <b>Daiki Kumakura</b>, Ryo Yamaguchi, Akane Hara, Shinji Nakaoka. Theoretical analysis of microbial dynamics considering functional labor. The 95th Annual Meeting of Japanese Society for Bacteriology, Japan. March 2022 (<b>invited</b>, oral, japanese)</p>

<p>&ensp;</p>

<p> <font size="4" color="#20b2aa"> <strong>Review articles and books</strong> </font> </p>

<p> <font size="4"> <b>2020</b> </font> </p>
<p>[1] Shinji Nakaoka, <b>Daiki Kumakura</b>. The role of mathematical sciences in preserving the gut environment. Experimental Medicine. 2020 Nov;38(18):3078-84. (japanese)</p>


<p>&ensp;</p>
<p>&ensp;</p>


<h2 id="CONTACT">CONTACT</h2>

<p> <font size="2">Laboratory of mathematical biology</font> </p>
<p> <font size="2">Transdisciplinary Life Science Course</font> </p>
<p> <font size="2">Division of Life Science, Graduate School of Life Science, Hokkaido University</font> </p>

<p>&ensp;</p>

<p> <font size="4" color="#20b2aa"> <strong>Current address :</strong> </font> </p>
<p> <font size="2"> Floor 7, Science Building No 2, Kita 10, Nishi 8, Kita-ku, Sapporo, Hokkaido 060-0810, Japan.</font> </p>

<p>&ensp;</p>

<p> <font size="4" color="#20b2aa"> <strong>Email :</strong> </font> </p>
<p> <font size="2"> <b>dkumakura [at] eis.hokudai.ac.jp</b> </font> </p>


<p>&ensp;</p>
<p>&ensp;</p>


<h2 id="LINK">LINK</h2>

<p>
  <b>・</b><a href="https://life.sci.hokudai.ac.jp/en/tl"> <font size="2"> <b>Hokkaido University</b> </font> </a>
</p>

<p>
  <b>・</b><a href="https://altair.sci.hokudai.ac.jp/infmcb/index-e.html"> <font size="2"> <b>Lab Homepage</b> </font> </a>
</p>

<p>
  <b>・</b><a href="https://qiita.com/kumalpha"> <font size="2"> <b>Qiita</b> </font> </a>
</p>

<p>
  <b>・</b><a href="https://github.com/DaikiKumakura"> <font size="2"> <b>GitHub</b> </font> </a>
</p>


<p>&ensp;</p>
<p>&ensp;</p>


<!--
<h2 id="ARTICLES">ARTICLES</h2>

<p>
  <b>・</b><a href="file:///Users/daikikumakura/Desktop/HOMEPAGE/article_1.html"> <font size="2"> <b>Notes on Empirical Dynamic Modeling</b> </font> </a>
</p>

<p>&ensp;</p>
<p>&ensp;</p>
-->

</div>
</div>
<div id="footer">© 2022 Daiki Kumakura</div>
</div></body>
</html>

